README.md

```diff
@@ -40,28 +40,28 @@ See [source](examples/example-shader/src/lib.rs) for full details.
+At Embark, we've been building our own new game engine from the ground up in Rust. We have previous experience in-house developing the [RLSL](https://github.com/MaikKlein/rlsl) prototype, and we have a team of excellent rendering engineers that are familiar with the problems in current shading languages both from games, game engines, and other industries. So, we believe we are uniquely positioned to attempt to solve this problem. 
 
-We want to streamline our own internal development with a single great language, build an open source graphics ecosystem and community, facilitate code-sharing between GPU and CPU, and most importantly: to enable our (future) users, and fellow developers, to more rapidly build great looking and engaging experiences.
+We want to streamline our own internal development with a single great language, build an open-source graphics ecosystem and community, facilitate code-sharing between GPU and CPU, and most importantly: to enable our (future) users, and fellow developers, to more rapidly build great looking and engaging experiences.
 
-If we do this project right, one wouldn't necessarily need an entire team of rendering engineers to build a good looking game, instead one would simply use a few of the existing open-source crates that provide the graphical effects needed to create the experience you're after. Instead of sharing and copy'n'pasting snippets of TAA code on forum posts, one could simply find and use the right crates from [crates.io](https://crates.io).
+If we do this project right, one wouldn't necessarily need an entire team of rendering engineers to build a good-looking game, instead one would simply use a few of the existing open-source crates that provide the graphical effects needed to create the experience you're after. Instead of sharing and copy 'n' pasting snippets of TAA code on forum posts, one could simply find and use the right crates from [crates.io](https://crates.io).
 
 ## Project scope
 
-The scope of this overall project is quite broad, but is in multiple stages
+The scope of this overall project is quite broad but is in multiple stages
 
 - `rustc` compiler backend to generate [SPIR-V], plugging in via `-Z codegen-backend`. 
   - This is the same mechanism that [rustc_codegen_cranelift](https://github.com/bjorn3/rustc_codegen_cranelift) and [rustc_codegen_gcc](https://github.com/antoyo/rustc_codegen_gcc) use.
-  - Currently only [SPIR-V] support is planned, [Vulkan](https://en.wikipedia.org/wiki/Vulkan_(API))'s open compiler target
-  - Possible a future version could suppport [DXIL](https://github.com/microsoft/DirectXShaderCompiler/blob/master/docs/DXIL.rst) (the target for DirectX) or [WGSL](https://github.com/gpuweb/gpuweb/tree/main/wgsl) (the WebGPU shading language that's bijective with SPIR-V)
+  - Currently, only [SPIR-V] support is planned, [Vulkan](https://en.wikipedia.org/wiki/Vulkan_(API))'s open compiler target
```

Comment:
Should just restructure this sentence to "Currently, support is planned only for SPIR-V, Vulkan's open compiler target"